src/net/FetchHttpClient.ts

```diff
@@ -11,6 +12,9 @@ const {HttpClient, HttpClientResponse} = require('./HttpClient');
  * node-fetch package (https://github.com/node-fetch/node-fetch).
  */
 class FetchHttpClient extends HttpClient {
+  _fetchFn: any;
+  _res: any;
```

Comment:
I'd prefer unknown if we don't know what's that or `Response` if it fits.

Reply:
`Response` fits!